accommodation/views.py

```diff
@@ -1 +1,131 @@
+<<<<<<< HEAD
+import json
+from django.shortcuts import render
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import Count, Avg
+from user.models      import User
+from .models          import  (
+    Theme,
+    AccommodationTheme, 
+    Accommodation, 
+    Host, 
+    RoomAmenity, 
+    RoomType,
+    RoomTypeAmenity,
+    AccommodationImage,
+    FreeService,
+    AccommodationService,
+    SharedFacility,
+    AccommodationFacility,
+    Dormitory,
+    AccommodationDormitory,
+    GeneralRoom,
+    AccommodationRoom,
+    Review,
+    AgeGroup,
+    TravelType,
+    Reservation,
+    Coupon,
+    UserCoupon,
+    AccommodationPayment,
+    PaymentMethod
+)
+
+
+class AccommodationDetailView(View):
+    def get(self, request, accommodation_id):
+        try:
+            limit  = int(request.GET.get("limit", "5"))
+            offset = int(request.GET.get("offset", "0"))
+            average_star_rating = Review.objects.filter(accommodation_id= accommodation_id).aggregate(Avg('star_rating'))["star_rating__avg"]
+            accommodation = Accommodation.objects.select_related(
+                'host'
+                ).prefetch_related(
+                'accommodation_service',
+                'accommodation_facility',
+                'accommodation_dormitory',
+                'accommodation_room',
+                'accommodationimage_set',
+                'theme_set',
+                'roomtype_set',
+                'review_set',
+                ).get(id=accommodation_id)
+
+            accommodation_detail = [
+                {
+                'id'                        : accommodation.id,
+                'name'                      : accommodation.name,
+                'description'               : accommodation.description,
+                'number_of_guest'           : accommodation.number_of_guest,
+                'check_in'                  : accommodation.check_in,
+                'check_out'                 : accommodation.check_out,
+                'address'                   : accommodation.address,
+                'is_immediate_confirmation' : accommodation.is_immediate_confirmation,
+                'image_url'                 : [ image.image_url for image in accommodation.accommodationimage_set.all() ],
+                'host'                      : [
+                    {
+                        'name'      : accommodation.host.name,
+                        'image_url' : accommodation.host.image_url
+                    }
+                ],
```

Comment:
여기서도 host 정보를 list로 만드는 이유가 있나요? 
```suggestion
                'host'                      : [
                    {
                        'name'      : accommodation.host.name,
                        'image_url' : accommodation.host.image_url
                    }
                ],
```